internal/bean/implement/mail_client.go

```diff
@@ -9,38 +9,41 @@ import (
 	"gopkg.in/gomail.v2"
 )
 
-type MailService struct {
+type MailCLient struct {
 	host     string
 	port     int
 	username string
 	password string
 	from     string
```

Comment:
why we need these attribute ? I think we only need the dialer since you only use these variables to init the dialer